src/components/Table/TableToolbarSearch/TableToolbarSearch.jsx

```diff
@@ -0,0 +1,101 @@
+/* istanbul ignore file */
+/* eslint-disable jsx-a11y/no-static-element-interactions */
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+
+import React, { useState, useEffect, useRef } from 'react';
+import PropTypes from 'prop-types';
+import { DataTable } from 'carbon-components-react';
+import classnames from 'classnames';
+
+import { settings } from '../../../constants/Settings';
+
+const { iotPrefix } = settings;
+
+const { TableToolbarSearch: CarbonTableToolbarSearch } = DataTable;
+
+const propTypes = {
+  defaultValue: PropTypes.string,
+  defaultExpanded: PropTypes.bool,
+  onChange: PropTypes.func,
+  onExpand: PropTypes.func,
+};
+
+const defaultProps = {
+  defaultValue: '',
+  defaultExpanded: false,
+  onChange: null,
+  onExpand: null,
+};
+
+/**
+ * ----------------------------- WARNING ----------------------------
+ * This is a temporary solution to fix a bug in Carbon 9,
+ * to be removed when we import Carbon 10
+ * ------------------------------------------------------------------
+ */
+const TableToolbarSearch = ({
+  defaultValue,
+  defaultExpanded,
+  onChange: onChangeProp,
```

Comment:
Looks like this is unneeded as the only time `onChange` is used, is when passing it to the `onChange` event handler